014_number_as_sum_of_squares.js

```diff
@@ -0,0 +1,22 @@
+//program to check if number can be expressed as sum of squares of two whole numbers
+const numberAsSumOfSquares = (number) => {
+    let result;
+    sumOfSquares = false;
+    for (i = 0; i <= number; i++) {
+        for (k = 0; k <= number; k++) {
+            result = (i * i + k * k);
+            if (result === number) {
+                sumOfSquares = true;
```

Comment:
Can you add the output as pull request description?

Reply:
ok.

Reply:
will update.

Reply:
I have added the output in the pull request description.